.github/workflows/build-jenkins-plugin.yml

```diff
@@ -0,0 +1,16 @@
+name: Build Jenkins Plugin
+
+on: [push]
+
+jobs:
```

Comment:
on tag, I think the workflow should fix the cersion according to the tag.
for PRs and branches, it can behave as is.

Reply:
Created a new workflow to build tags and the tag will be appended to the revision which combinely forms the plugin version.

eg: On adding tag BIT-0.0.1 to the Github repo the version of plugin appears as 2.9.12-BIT-0.0.1